### DISCUSSION TITLE
Global Shortcut

### DISCUSSION THREAD

**Discussion by lk23lekkladf2** [rxhanson/Rectangle#527] [Ideas]
Does Rectangle support an entry point shortcut? In other apps, [Divvy](https://mizage.com/divvy/) for example, this is called **Global Shortcut**. This Global Shortcut launches the application in a mode where it's listening for app specific shortcuts.

Sounds odd, but an example explains it better.  Assuming `⌃⌥R` is the Rectangle app shortcut entry point one would do `⌃⌥R` and then `C` to center instead of `⌃⌥ C`. Similarly, one would do `⌃⌥R` and then `F` for the center third instead of `⌃⌥ F`

Seems like an unnecessary indirection shortcut, however, one key advantage is that it allows you to set shortcuts that would otherwise conflict with the rest of the system.

Have you all considered this functionality? It doesn't appear to be supported at the moment.

Thanks,
Jorge.

**Comment by rxhanson** [comment]
Thanks for the idea. I have thought about this one before, and had one request for it previously. It's not something that I would use personally, so it's not something I have motivation to add in myself. But, I can see the appeal, and don't see a reason to turn down a good PR with it.
